recipes-kernel/linux/linux-toradex_5.4-2.3.x.bb

```diff
@@ -0,0 +1,32 @@
+require recipes-kernel/linux/linux-imx.inc
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+
+DEPENDS += "lzop-native bc-native"
+
+SUMMARY = "Linux kernel for Toradex Freescale i.MX based modules"
+
+SRC_URI = "git://git.toradex.com/linux-toradex.git;protocol=git;branch=${SRCBRANCH}"
+
+# Load USB functions configurable through configfs (CONFIG_USB_CONFIGFS)
+KERNEL_MODULE_AUTOLOAD += "${@bb.utils.contains('COMBINED_FEATURES', 'usbgadget', ' libcomposite', '',d)}"
```

Comment:
Why this explicit autoload is need?

Reply:
libcomposite is not loaded implicitely.

With the module loaded one can configure USB gadget functionality without having to integrate the modprobe of libcomposite.

I like to keep that in.

Stefan created some nice docu on how to use USB gadget.
https://developer.toradex.com/knowledge-base/usb-device-mode-linux#NXPFreescale_VybridiMX7iMX6ULLiMX6iMX8iMX8X_based_Modules

Reply:
This seems wrong and seems like it belongs to the `configfs` configuration script (as it'll be needed anyway). I won't block the merging as it is in your Linux kernel recipe but I see no good reason for it.

Reply:
I drop it and move it to the systemd service we use to configure the USB gadget.